ccmlib/scylla_node.py

```diff
@@ -570,11 +570,12 @@ def process_opts(opts):
                         opts_i += 1
                     val = ' '.join(vals)
                 if key not in ext_args and not key.startswith("--scylla-manager"):
-                    ext_args[key] = val
+                    ext_args[args_translation_map.get(key, key)] = val
```

Comment:
i'd suggest normalize `key` even before the `if key not in ext_args and not key.startswith("--scylla-manager")` check, and do this unconditially. like:
```py
key = args_translation_map.get(key, key)
if key not in ext_args and not key.startswith("--scylla-manager"):
    ext_args[key] = val
```

for better readaiblity.